CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ available [on GitHub][2].
 - [#31](https://github.com/chshersh/zbg/pull/31):
   Command `zbg push -f` now uses git's `--force-with-lease` because it's a safer default
   (by [@kephas])
+-[#10](https://github.com/chshersh/zbg/issues/10):
+  Changed `fetch_main_branch` to get the base name using `Core.Filename.basename` rather than reading the output of the `basename` command.
+  (by [@LeedsJohn])
```

Comment:
Could you also add your username to the Contributors section in alphabetical order since this is your first contribution?

https://github.com/chshersh/zbg/blob/3e5174128dfd5728271c1562ae4bb8d22e501723/CHANGELOG.md#L51-L59

Reply:
Just fixed - thanks!